src/Veldrid/FormatHelpers.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Veldrid
 {
-    internal static class FormatHelpers
+    public static class FormatHelpers 
```

Comment:
I found some code in the samples repo that duplicated the format->size conversion here for only one or two formats.   Since sizing and packing data is an important part of the binding process, I figured it'd be best to push shift the maintenance burden into the library and expose the canonical implementation of these conversions to anyone who needs to use it.

Reply:
Moved `GetSizeInBytes` to new public class `FormatSizeHelpers` and made `FormatHelpers` internal again.